File: NetFramework/XHtmlEditor/CS/RichTextEditor/AppMenuTabs/PrintTab.cs
using RichTextEditor.Properties;
using System;
using System.Drawing;
using System.Windows.Forms;

namespace RichTextEditor.AppMenuTabs
{
    public partial class PrintTab : UserControl
    {
        #region Ctor
        public PrintTab()
        {
            InitializeComponent();

            lblCaption.Text = GetElementText(Resources.PrintTab_txt);
            SetButtonSettings(btnPrint, "Print");
            SetButtonSettings(btnQPrint, "QuickPrint");

            c1PrintPreviewControl1.TopToolStripPanelVisible = false;            
        }
        #endregion

        #region Properties
        public C1TextEditorRibbon Ribbon
        {
            get;
            set;
        }
        #endregion

        #region Event handlers
        private void btnQPrint_Click(object sender, EventArgs e)
        {
            if (Ribbon != null)
                Ribbon.PrintDocument(false, false);
        }

        private void btnPrint_Click(object sender, EventArgs e)
        {
            if (Ribbon != null)
                Ribbon.PrintDocument(true, false);
        }

        #endregion

        #region Methods
        private void SetButtonSettings(Button btn, string prefix)
        {
            btn.Image = (Image)Resources.ResourceManager.GetObject(prefix + "_large", Resources.Culture);

            btn.TextImageRelation = TextImageRelation.ImageAboveText;
            btn.ImageAlign = System.Drawing.ContentAlignment.MiddleCenter;
            btn.TextAlign = System.Drawing.ContentAlignment.BottomCenter;
            // text resources
            string res = (string)Resources.ResourceManager.GetObject(prefix + "_txt", Resources.Culture);
            string[] textParts = res != null
                ? res.Split('\n')
                : new string[] { prefix };

            // Text
            string text = textParts[0].Trim();
            if (!string.IsNullOrEmpty(text))
            {
                btn.Text = text;
            }
            // ToolTip: text and body
            if (textParts.Length > 1)
            {
                if (!textParts[1].StartsWith("*"))
                {
                    var toolTip = new C1.Win.C1SuperTooltip.C1SuperTooltip();
                    string tipTitle = textParts[1].Trim();
                    string tipBody = string.Join("<br/>", textParts, 2, textParts.Length - 2).Trim();
                    toolTip.SetToolTip(btn, string.IsNullOrEmpty(tipBody)
                        ? tipTitle
                        : string.Format(
                            "<b>{0}</b><br/>" +
                            "<div style='margin-left:12'>{1}</div>",
                            tipTitle, tipBody));
                }
            }
        }
        
        private string GetElementText(string text)
        {
            string[] textParts = text != null
                ? text.Split('\n')
                : new string[] { string.Empty };

            return textParts[0].Trim();
        }

        internal void SetDocument()
        {
            c1PrintPreviewControl1.Document = Ribbon.Editor.PrintDocument;
        }

        #endregion
    }
}

File: NetFramework/XHtmlEditor/VB/RichTextEditor/BackstageTabs/PrintTab.vb

Imports System.Drawing
Imports System.Windows.Forms
Imports C1.Win.SuperTooltip
Imports RichTextEditor.RichTextEditor

Partial Public Class PrintTab
    Inherits UserControl

    Public Sub New()
        InitializeComponent()
        lblCaption.Text = GetElementText(Resources.PrintTab_txt)
        SetButtonSettings(btnPrint, "Print")
        SetButtonSettings(btnQPrint, "QuickPrint")
        c1PrintPreviewControl1.TopToolStripPanelVisible = False
    End Sub

    Public Property Ribbon As C1TextEditorRibbon

    Private Sub btnQPrint_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnQPrint.Click
        If Ribbon IsNot Nothing Then Ribbon.PrintDocument(False, False)
    End Sub

    Private Sub btnPrint_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnPrint.Click
        If Ribbon IsNot Nothing Then Ribbon.PrintDocument(True, False)
    End Sub

    Private Sub SetButtonSettings(ByVal btn As Button, ByVal prefix As String)
        btn.Image = CType(Resources.ResourceManager.GetObject(prefix & "_large", Resources.Culture), Image)
        btn.TextImageRelation = TextImageRelation.ImageAboveText
        btn.ImageAlign = System.Drawing.ContentAlignment.MiddleCenter
        btn.TextAlign = System.Drawing.ContentAlignment.BottomCenter
        Dim res As String = CStr(Resources.ResourceManager.GetObject(prefix & "_txt", Resources.Culture))
        Dim textParts As String() = If(res IsNot Nothing, res.Split(vbLf), New String() {prefix})
        Dim text As String = textParts(0).Trim()
        If Not String.IsNullOrEmpty(text) Then
            btn.Text = text
        End If

        If textParts.Length > 1 Then
            If Not textParts(1).StartsWith("*") Then
                Dim toolTip As C1SuperTooltip = New C1.Win.SuperTooltip.C1SuperTooltip()
                Dim tipTitle As String = textParts(1).Trim()
                Dim tipBody As String = String.Join("<br/>", textParts, 2, textParts.Length - 2).Trim()
                toolTip.SetToolTip(btn, If(String.IsNullOrEmpty(tipBody), tipTitle, String.Format("<b>{0}</b><br/>" & "<div style='margin-left:12'>{1}</div>", tipTitle, tipBody)))
            End If
        End If
    End Sub

    Private Function GetElementText(ByVal text As String) As String
        Dim textParts As String() = If(text IsNot Nothing, text.Split(vbLf), New String() {String.Empty})
        Return textParts(0).Trim()
    End Function

    Friend Sub SetDocument()
        C1PrintPreviewControl1.Document = Ribbon.Editor.PrintDocument
    End Sub
End Class